### ISSUE TITLE
more of a question than a issue 

### ISSUE THREAD

**Issue by maxlink89** [ekrividus/autoMB#13]
is it possable to add a UI to automb to show what is selected for it ?

**Comment by maxlink89** [comment]
sorta like what you have done with autosc

**Comment by ekrividus** [comment]
Yeah that was on my (so far only in my head) todo list before I took my winter recess.
